<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stiai ca aceasta tara are...?</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>777</width>
          <height>557</height>
         </rect>
        </property>
        <widget class="QPushButton" name="pushButton_export">
         <property name="geometry">
          <rect>
           <x>630</x>
           <y>10</y>
           <width>131</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Exporta info</string>
         </property>
        </widget>
        <widget class="QListWidget" name="QListWidget_countries">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>361</width>
           <height>351</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
        <widget class="QLabel" name="label_capitala">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>260</y>
           <width>251</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Capitala</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_populatia">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>300</y>
           <width>251</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Populatie</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_continent">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>220</y>
           <width>251</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Continent</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_moneda">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>380</y>
           <width>251</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Moneda</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_suprafata">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>340</y>
           <width>251</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Suprafata</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_wiki_page">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Pagina wiki</string>
         </property>
        </widget>
        <widget class="QListWidget" name="qListWidget_judete">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>130</y>
           <width>361</width>
           <height>91</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_timezone">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>420</y>
           <width>251</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Time zone</string>
         </property>
        </widget>
        <widget class="QCommandLinkButton" name="commandLinkButton_wiki">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>40</y>
           <width>185</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Go To Wikipedia</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_continent_2">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>220</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Continent:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_capitala_2">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>260</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Capitala:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_populatia_2">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>300</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Populatie:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_suprafata_2">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>340</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Suprafata:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_moneda_2">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>380</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Moneda:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_timezone_2">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>420</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Time zone:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_judete">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>100</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Provincii:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="textEdit_search">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>361</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="placeholderText">
          <string>Cauta o tara</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
